src/Extension/HeadingPermalink/HeadingPermalinkProcessor.php

```diff
@@ -58,11 +58,13 @@ public function __invoke(DocumentParsedEvent $e): void
     {
         $this->useNormalizerFromConfigurationIfProvided();
 
+        $levels = $this->config->get('heading_permalink/levels', [1, 2, 3, 4, 5, 6]);
```

Comment:
`/** @var int[] */`

Reply:
The ToC extension already has simlar configuration:

```
        'min_heading_level' => 1,
        'max_heading_level' => 6,
```

Can we use the same config min/max here to avoid introducing confusion (trying to remember which is which)?

Or, introduce this new array of `levels` to the ToC extension and deprecate the min/max approach?

Reply:
Good catch.  This has been changed via 2aa08dc3.